src/common/components/paymentMethods/editPaymentMethod/editPaymentMethod.modal.component.js

```diff
@@ -0,0 +1,57 @@
+import angular from 'angular';
+
+import template from './editPaymentMethod.modal.tpl';
+
+import paymentMethodDisplay from 'common/components/paymentMethods/paymentMethodDisplay.component';
+
+import loadingOverlay from 'common/components/loadingOverlay/loadingOverlay.component.js';
+
+import bankAccountForm from '../bankAccountForm/bankAccountForm.component';
+import creditCardForm from '../creditCardForm/creditCardForm.component';
+
+let componentName = 'editPaymentMethodModal';
+
+class editPaymentMethodModalController{
```

Comment:
@OzzieOrca, I was also thinking about reusing your modal, but was a little hesitant because I'm also a little afraid it can get a little too messy. I might be wrong. Let's keep it that way and after all is set and done with adding and editing we can take a look at the whole picture and make a decision about combining vs keeping separate.